src/protocol/commit_strategy.md

```diff
@@ -1,21 +1,24 @@
-# Commit Strategy
-When the `Prover` generates authenticated commitments to the plaintext of the transcript, he can choose which portion(s) of the plaintext to commit to. This dictates the portion(s) that can be selectively disclosed later to the application-specific `Verifier`. This section discusses different commit strategies, as well as their associated costs.
+When the `Prover` generates authenticated commitments to the plaintext of the transcript, it is possible to choose which range(s) of the plaintext to commit to. This dictates the range(s) that can be selectively disclosed later to the application-specific `Verifier`. This section discusses different commit strategies, as well as their associated costs.
 
 ## Strategy
-Given a transcript plaintext of `N` bytes long, the `Prover` can choose from a variety of commit strategies. For example, he can perform one commit to the portion that corresponds to the response body. He can also perform `N` commits to every byte of the transcript. The details of various commit strategies are discussed in the table below.
+Depending on the application, different commit strategies can be chosen. One can commit to everything at once, to each byte individually, or any selection of ranges. There is no significant computational impact when you choose a large number of commitments, but there is a linear storage cost increase.
 
-| # | Strategy | Description | Selective Disclosure | Cost | Usage |
-| - | -------- | ----------- | -------------------- | ---- | ----- |
-| 1 | Commit to custom portion(s) | Only commit to the portion(s) that need to be selectively disclosed later | Cannot selectively disclose anything other than the committed portion(s) | Small artefact size if only a few portions committed | Suitable when only a single (or a few) selective disclosure pattern is needed |
-| 2 | Commit to HTTP objects | Commit to portions that correspond to all common HTTP objects in both request and response, e.g. every header's key and value, every key and value in JSON response body | Flexible to selectively disclose different HTTP objects of transcript at different times | Larger artefact size than strategy #1 | Ideal for most use cases — the default strategy used in the repository's [example](https://github.com/tlsnotary/tlsn/blob/4d5102b6e141ecb84b8a835604be1d285ae6eaa5/crates/examples/attestation/prove.rs#L99) |
-| 3 | Commit to each byte | One commit for each byte, resulting in `N` portions committed | Maximum flexibility as any portion of the transcript can be selectively disclosed at different times | Largest artefact size among all strategies | Suitable when needed to support selective disclosure on many arbitrary portions beyond common HTTP objects |
+TLSNotary offers a default strategy that commits to ranges corresponding with HTTP objects. This results in a good balance between *reveal flexibility* and *storage size* for most use cases.
```

Comment:
*reveal flexibility* -> *selective disclosure flexibility*? just to keep the terms consistent